ansible/doc/README.testbed.VsSetup.md

```diff
@@ -166,6 +170,8 @@ $ ./testbed-cli.sh -t vtestbed.csv -m veos.vtb add-topo vms-kvm-t0 password.txt
 $ ./testbed-cli.sh -t vtestbed.csv -m veos.vtb -k ceos add-topo vms-kvm-t0 password.txt
 ```
 
+  - please note: Here "password.txt" is the ansible vault password file name/path. Ansible allows user use ansible vault to encrypt password files. By default, this shell script require a password file. If you are not using ansible vault, just create a file with a dummy pasword and pass the filename to the command line. The file name and location is created and maintained by user.
```

Comment:
s/please/Please/
s/ansible vault/Ansible Vault/ (multiple occurrences)
s/By default, this shell script require a password file.By default, this shell script requires a password file./
s/Ansible allows user use/Ansible allows user to use/
s/pasword/password/

Reply:
Thanks, I fixed the grammar here and in the earlier description of password.txt in the vEOS section (Setup Arista VMs in the server)